module/localisation/RobotLocalisation/src/RobotModel.hpp

```diff
@@ -69,6 +69,9 @@ namespace module::localisation {
             StateVec(const Eigen::MatrixBase<OtherDerived>& state)
```

Comment:
minor: Probably best to specify this exact dimension here too
```suggestion
            StateVec(const Eigen::Matrix<Scalar, size, 1>& state)
```